profiler/profile_test.go

```diff
@@ -171,7 +171,7 @@ main;bar 0 0 8 16
 	})
 
 	t.Run("cpu", func(t *testing.T) {
-		p, err := unstartedProfiler(CPUDuration(10 * time.Millisecond))
+		p, err := unstartedProfiler(CPUDuration(10*time.Millisecond), WithPeriod(10*time.Millisecond))
```

Comment:
This is necessary, or else the profile waits `default period - 10 milliseconds`, i.e. almost 60 seconds.